data/restli-csharp/datagenerator/src/test/GeneratedDataTemplate/com/linkedin/restli/datagenerator/integration/EnumInMapTest.cs

```diff
@@ -0,0 +1,36 @@
+// DO NOT EDIT - THIS FILE IS GENERATED BY restli-csharp
+// Generated from com/linkedin/restli/datagenerator/integration/MapTest.pdsc
+
+namespace com.linkedin.restli.datagenerator.integration
+{
+  
+  public class EnumInMapTest {
+    public enum Value
+    {
+
+      A,
+
+      B,
+
+      @UNKNOWN
+    }
+
+    public Value? value { get; }
+
+    public EnumInMapTest(string data)
```

Comment:
Besides generating this constructor taking a String, we should also generate a constructor to take a data of type `EnumInMapTest.Value`